packaging/cmake/Modules/FindZstd.cmake

```diff
@@ -0,0 +1,28 @@
+# https://github.com/facebook/folly/blob/master/CMake/FindZstd.cmake
+#
```

Comment:
Need to include the folly license in this directory as LICENSE.FindZstd

Reply:
Hi! @edenhill Thanks for your review! I have added a license file for `FindZstd.cmake`: https://github.com/myd7349/librdkafka/blob/win32-zstd-static-lib/LICENSE.FindZstd
BTW, I cut these lines off: https://github.com/facebook/folly/blob/master/LICENSE#L180-L200